src/main/java/uk/gov/hmcts/reform/hmc/data/PartyRelationshipDetailsEntity.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.hmc.data;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Table(name = "party_relationship_details")
+@Entity
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PartyRelationshipDetailsEntity implements Serializable {
```

Comment:
Does this need some foreign key/table link ?

Reply:
Fixed by @danlysiak in https://github.com/hmcts/hmc-cft-hearing-service/pull/153/commits/50b833069211b0402b67717fae973ed563286907